src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/Select.js

```diff
@@ -118,6 +189,70 @@ class Select<T> extends React.Component<Props<T>> {
 
     handleOptionListClose = this.closeOptionList;
 
+    @action requestFocus = (elementIndex: number) => {
+        if (!this.buttonRefsByIndex.has(elementIndex)) {
+            return;
+        }
+
+        this.focusedElementIndex = elementIndex;
+        const ref = this.buttonRefsByIndex.get(elementIndex);
+
+        if (ref) {
+            ref.focus();
+        }
+    };
+
+    handleRequestFocus = (elementIndex: number) => () => {
+        this.requestFocus(elementIndex);
+    };
+
+    @action handleKeyDown = (event: KeyboardEvent) => {
```

Comment:
i think we need to handle `Enter` here too. otherwise i am not able to submit my selection

Reply:
also the native select is opened when pressing `ArrowUp` or `ArrowDown` while it is focused. should we implement that too?
https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_select

Reply:
fixed